components/profile/shared/lists/donationList/ImpactCache.ts

```diff
@@ -0,0 +1,151 @@
+export {};
```

Comment:
Denne kan slettes

Reply:
This file no longer exists, ok if I resolve comment?